src/ResourceManager/Insights/Commands.Insights/Alerts/RemoveAzureRmAlertRuleCommand.cs

```diff
@@ -15,13 +15,14 @@
 using Microsoft.Azure.Management.Insights;
 using System.Collections.Generic;
 using System.Management.Automation;
+using Microsoft.Azure.Management.Insights.Models;
 
 namespace Microsoft.Azure.Commands.Insights.Alerts
 {
     /// <summary>
     /// Remove an Alert rule
     /// </summary>
-    [Cmdlet(VerbsCommon.Remove, "AzureRmAlertRule"), OutputType(typeof(List<AzureOperationResponse>))]
+    [Cmdlet(VerbsCommon.Remove, "AzureRmAlertRule"), OutputType(typeof(PSObject))]
```

Comment:
@gucalder this is a breaking change; if I had a script that did the following
```powershell
$foo = Remove-AzureRmAlertRule -ResourceGroup $ResourceGroup -Name $Name
```
and I tried to do anything with `$foo`, it will no longer work